frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineMetricsQuickstart.tsx

```diff
@@ -61,7 +67,13 @@ const PipelineMetricsQuickstart: React.FC = () => {
               <Alert
                 variant="info"
```

Comment:
Can we also show this as a warning if its unsupported?

```suggestion
                variant={metricsLevel === PipelineMetricsLevel.UNSUPPORTED_LEVEL ? "warning" : "info"}
```

(There might be some AlertVariant consts for warning and info as well.